src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -781,48 +781,49 @@ void Terminal::_AdjustCursorPosition(const COORD proposedPosition)
     auto proposedCursorPosition = proposedPosition;
     auto& cursor = _buffer->GetCursor();
     const Viewport bufferSize = _buffer->GetSize();
-    bool notifyScroll = false;
 
     // If we're about to scroll past the bottom of the buffer, instead cycle the
     // buffer.
-    // GH#5540 - Make sure this is a positive number. We can't create a
-    // negative number of new rows.
-    const auto newRows = std::max(0, proposedCursorPosition.Y - bufferSize.Height() + 1);
-    if (newRows > 0)
+    SHORT fakeScroll = 0;

```

Comment:
This would probably be clearer if the variable was `scrollDelta` instead of `fakeScroll`

Reply:
Isn't `scrollDelta` is too generic? Because this variable is only set when there is a vertical scroll and the view port will not change after the vertical scroll, which is only the case after the buffer is full and starts to cycle.

Reply:
I think I'm okay with it... or, `rowsPushedOffTopOfBuffer` is a little less of a mouthful.